resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,426 @@
-                <tr>
-                  <td>Flight Maps:</td>
-                  <td><select name="maps">
-                      <option value="detail">Detailed</option>
-                      <option value="simple">Simple</option>
-                      <option value="none">None</option>
-                    </select></td>
-                </tr>
-              </table>
-            </div>
-          </div>
-        </div>
+{{-- SIMPLE ERROR PROTECTION; If the Aircraft_ID is not passed stop procession code dispay an error message --}}
+@if(app('request')->input('aircraft_id'))
+
+@php 
+	$SelectedAircraft = app('request')->input('aircraft_id') ;
+	$AircraftDetails = DB::table('aircraft')->select('registration', 'icao', 'iata', 'subfleet_id')->where('id', $SelectedAircraft)->get() ;
+@endphp
+
+{{-- Get Aircraft Details and Apply ICAO Type Corrections For SimBrief --}}
+@foreach($AircraftDetails as $AcDetails)
+	@php 
+		$SimBriefType = $AcDetails->icao ;
+		$SubFLid = $AcDetails->subfleet_id ;
+		if($AcDetails->icao == 'A20N') { $SimBriefType = 'A320' ; }
+		if($AcDetails->icao == 'A21N') { $SimBriefType = 'A321' ; }
+		if($AcDetails->icao == 'B77L') { $SimBriefType = 'B77F' ; }
+		if($AcDetails->icao == 'B773') { $SimBriefType = 'B77W' ; }
+		if($AcDetails->icao == 'E35L') { $SimBriefType = 'E135' ; }
+	@endphp
+@endforeach
 
-        <input type="hidden" name="airline" value="{{ $flight->airline->icao }}">
-        <input type="hidden" name="fltnum" value="{{ $flight->flight_number }}">
-        <input type="hidden" name="date" value="01JAN14">
-        <input type="hidden" name="deph" value="12">
-        <input type="hidden" name="depm" value="30">
-        <input type="hidden" name="steh" value="2">
-        <input type="hidden" name="stem" value="15">
-        {{--<input type="hidden" name="reg" value="N123SB">
-        <input type="hidden" name="selcal" value="GR-FS">--}}
-        <input type="hidden" name="planformat" value="lido">
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-12">
-        <div class="float-right">
-          <div class="form-group">
-            <input type="button"
-                   onclick="simbriefsubmit('{{ $flight->id }}', '{{ url(route('frontend.simbrief.briefing', [''])) }}');"
-                   class="btn btn-outline-primary"
-                   value="Generate">
-          </div>
-        </div>
-      </div>
-    </div>
-  </form>
+@php if($flight->alt_airport_id) { $ALTN = $flight->alt_airport_id ; } else { $ALTN = 'AUTO' ; } @endphp
+
+{{-- Define The Random Load Factor, Get Max Capacity of Selected SubFleet and Generate Load --}}
+@php
+	$FKMin = $flight->load_factor - $flight->load_factor_variance ;
+	$FKMax = $flight->load_factor + $flight->load_factor_variance ;
+	if($FKMin < 1) { $FKMin = 1 ; }	
+	if($FKMax > 100) { $FKMax = 100 ; }
+	$FKRandomLoad = rand($FKMin, $FKMax);
+	$GetMaxCap = DB::table('subfleet_fare')->where('subfleet_id', $SubFLid)->sum('capacity') ;
```

Comment:
This logic should also move into the controller and get passed in

Reply:
Removed the GetMaxCap completely